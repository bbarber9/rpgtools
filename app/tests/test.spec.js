import React from 'react';

describe('test module 1', () => {
	test('test1', () => {
		expect({attr: 1}).toMatchSnapshot()
	});
	test('test2', () => {
		expect({attr: 2}).toMatchSnapshot()
	});
});