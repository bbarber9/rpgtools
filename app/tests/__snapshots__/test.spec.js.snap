// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test module 1 test1 1`] = `
Object {
  "attr": 1,
}
`;

exports[`test module 1 test2 1`] = `
Object {
  "attr": 2,
}
`;
